it/docker/docker-compose.yml

```diff
@@ -22,50 +20,57 @@ services:
         -Dscheduler.topics.avro.0=avro-schedules
         -Dscheduler.topics.json.0=json-schedules
     depends_on:
-      kafka:
-        condition: service_started
-
-  zookeeper:
-    image: confluentinc/cp-zookeeper:7.5.1
-    environment:
-      ZOOKEEPER_CLIENT_PORT: '2181'
+      init-topics:
+        condition: service_completed_successfully
 
   kafka:
-    image: confluentinc/cp-kafka:7.5.1
+    image: bitnami/kafka:latest
     ports:
-      - "${KAFKA_PORT}:${KAFKA_PORT}"
+      - "9092:9092"
+      - "9094:9094"
     environment:
-      KAFKA_ZOOKEEPER_CONNECT: zookeeper:2181
-      KAFKA_OFFSETS_TOPIC_REPLICATION_FACTOR: '1'
-      KAFKA_ADVERTISED_LISTENERS: PLAINTEXT://kafka:9092,OUTSIDE://localhost:${KAFKA_PORT}
-      KAFKA_LISTENER_SECURITY_PROTOCOL_MAP: PLAINTEXT:PLAINTEXT,OUTSIDE:PLAINTEXT
+      - KAFKA_CFG_NODE_ID=0
+      - KAFKA_CFG_PROCESS_ROLES=controller,broker
+      - KAFKA_CFG_LISTENERS=PLAINTEXT://:9092,CONTROLLER://:9093,EXTERNAL://:9094
+      - KAFKA_CFG_ADVERTISED_LISTENERS=PLAINTEXT://kafka:9092,EXTERNAL://localhost:9094
+      - KAFKA_CFG_LISTENER_SECURITY_PROTOCOL_MAP=CONTROLLER:PLAINTEXT,EXTERNAL:PLAINTEXT,PLAINTEXT:PLAINTEXT
+      - KAFKA_CFG_CONTROLLER_QUORUM_VOTERS=0@kafka:9093
+      - KAFKA_CFG_CONTROLLER_LISTENER_NAMES=CONTROLLER
```

Comment:
Can these stay as
```yaml
KAFKA_CFG_NODE_ID: 0
```
vs
```yaml
- KAFKA_CFG_NODE_ID=0
```
As it keeps the same style as elsewhere in this file

Reply:
https://github.com/sky-uk/kafka-message-scheduler/pull/398/commits/b8bbf4c50bbd0b1e8e86ad9b305e80c0bb86c58f